pyHS100/smartdevice.py

```diff
@@ -127,6 +134,17 @@ def _query_helper(self,
 
         return result
 
+    def _index_to_id(self, index: int) -> str:
```

Comment:
Is this really necessary? Given that the library is developer-oriented, it could be handled in the CLI, and whoever's using it to implement this themselves.

Reply:
Either way is fine but just seemed better to always allow passing the index instead of child id.